scripts/tests/test_prepare_infra_k8s_inputs.py

```
@@ -0,0 +1,124 @@
+    assert inputs.node_pools[0]["name"] == "default"
+
+
+def test_resolve_all_inputs_rejects_invalid_cluster(monkeypatch: pytest.MonkeyPatch, tmp_path: Path) -> None:
+    env = _base_env(tmp_path)
+    env["INPUT_CLUSTER_NAME"] = "Invalid_Name"
+    for key, value in env.items():
+        monkeypatch.setenv(key, value)
+
+    with pytest.raises(ValueError, match="cluster_name"):
+        raw_values = dict.fromkeys(RawInputs.__annotations__, None)
+        _resolve_all_inputs(RawInputs(**raw_values))
```

Ruff (0.14.13)

`pytest.raises()` block should contain a single simple statement

(PT012)